### PR TITLE
feat(i18n): add zh-CN i18n for external page

### PR DESCRIPTION
**Pull request by pionxzh**
### Description

This PR adds the `zh-CN` translation for the external page to make zh-cn 100% translated.

### CODE REVIEW

File: src/renderer/src/@types/default-resource.ts [hunk 1]
```diff
@@ -27,8 +31,12 @@ export const defaultResources = {
     shortcuts: shortcuts_en,
   },
   "zh-CN": {
+    app: zhCN,
     lang: lang_zhCN,
     common: common_zhCN,
+    external: external_zhCN,
```
Comment (Innei) [inline src/renderer/src/@types/default-resource.ts:37]: Remove this. DONT MODIFY THIS FILE

**Review by Innei — CHANGES_REQUESTED** [review]

File: src/renderer/src/@types/default-resource.ts (hunk 1 above)
Comment (pionxzh) [reply src/renderer/src/@types/default-resource.ts:37]: This one doesn't need to be translated?

File: src/renderer/src/@types/default-resource.ts [hunk 2]
```diff
@@ -27,8 +31,12 @@ export const defaultResources = {
     shortcuts: shortcuts_en,
   },
   "zh-CN": {
+    app: zhCN,
```
Comment (Innei) [inline src/renderer/src/@types/default-resource.ts:34]: Remove this

File: src/renderer/src/@types/default-resource.ts [hunk 3]
```diff
@@ -27,8 +31,12 @@ export const defaultResources = {
...
     lang: lang_zhCN,
     common: common_zhCN,
+    external: external_zhCN,
+    settings: settings_zhCN,
+    shortcuts: shortcuts_zhCN,
```
Comment (Innei) [inline src/renderer/src/@types/default-resource.ts:39]: Remove this

File: src/renderer/src/@types/default-resource.ts [hunk 4]
```diff
@@ -27,8 +31,12 @@ export const defaultResources = {
     shortcuts: shortcuts_en,
   },
   "zh-CN": {
+    app: zhCN,
     lang: lang_zhCN,
     common: common_zhCN,
+    external: external_zhCN,
+    settings: settings_zhCN,
```
Comment (Innei) [inline src/renderer/src/@types/default-resource.ts:38]: Remove this

**Review by Innei — CHANGES_REQUESTED** [review]

File: locales/external/zh-CN.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
+  "feed": {
+    "follow_one": "关注",
+    "follow_other": "关注者",
+    "followsAndReads": "{{subscriptionCount}} 个{{subscriptionNoun}}，在 {{appName}} 上有 {{readCount}} 个{{readNoun}}",
+    "read_one": "阅读",
+    "read_other": "阅读者"
```
Comment (Innei) [inline locales/external/zh-CN.json:7]: "read_other": "阅读"

**Review by Innei — CHANGES_REQUESTED** [review]

File: locales/external/zh-CN.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
+{
+  "feed": {
+    "follow_one": "关注",
```
Comment (Innei) [inline locales/external/zh-CN.json:3]: "follow_one": "关注者",

**Review by Innei — CHANGES_REQUESTED** [review]

File: locales/external/zh-CN.json (hunk 5 above)
Comment (pionxzh) [reply locales/external/zh-CN.json]: 这个我想问下 是应该翻为 "阅读" 还是 "文章"

Comment (pionxzh) [reply locales/external/zh-CN.json]: follow 支持多种类型，所以不知道用 "文章" 是不是合适的，这样问是因为 "阅读" 有点怪

**Review by Innei — APPROVED** [review]
Thank you for your contributions

**Pull request merged**